### DISCUSSION TITLE
临时通知！

### DISCUSSION THREAD

**Discussion by hao123nevergiveup** [share121/inter-knot#329] [其他]
up临时更新了绳网协约，详细视频见
https://www.bilibili.com/video/BV1GJ411x7h7

![niko1](https://github.com/user-attachments/assets/8503c0e4-bb6d-4747-a0b8-f58b16cfb622)

**Comment by Phaeton9528** [comment]
很详细的协约，确实不能什么都在这里乱发，建议大家都仔细看看

**Comment by magialord** [comment]
对，为了绳网正常运行，大家一定要仔细阅读

**Comment by biliqide** [comment]
一不小心就触发连招了😅 #

**Comment by tiang-gong** [comment]
110

**Comment by Yleoooooo** [comment]
到此一游，留个脚印

**Comment by EllenJoe14** [comment]
你 被 骗 了

**Comment by Yleoooooo** [comment]
到此一游，留个脚印

**Comment by HORNDAY** [comment]
up主说的太对了，眼泪瞬间就下来了：）

**Comment by CALLUM873** [comment]
说的太好了qwq

**Comment by yyxzz808** [comment]
？欺骗我的感情

**Comment by yoshizawa-kasumi** [comment]
好好好，不得不说，up说的太对了，建议所有人都看

**Comment by starfox11645** [comment]
泪目了，很难不智齿

**Comment by RGthx** [comment]
你是懂……节目效果的

**Comment by wugan-bot156** [comment]
挺及时的协约，建议后来的都看看，大家一起为绳网的建设做出贡献

**Comment by wwy2333** [comment]
这个委托我接了！

**Comment by waricly** [comment]
协约十分详细，而且人性化，真心建议看看，绝对不会被骗！

**Comment by ink808** [comment]
UP主声泪俱下，有理有据，令人印象深刻(o^^o)♪

**Reply by ink808** [reply]
以骸都不信

**Comment by WUWUWUxi** [comment]
上当了

**Comment by lllliufeng** [comment]
收到

**Comment by zhangyj0523** [comment]
不用点进去就知道是什么力

**Comment by udigjbx114514** [comment]
能被骗的一看就是没有下载国家反诈APP，不像我
![img1919094290_1722168832644](https://github.com/user-attachments/assets/c83141d0-2c90-4c08-8091-28f85d4788a8)
下了也没用

**Comment by Mnpolize** [comment]
谢谢，已经变成遵纪守法好公民了

**Comment by alonea13** [comment]
_**这才是正版绳网协约**_
<a style="--icon-width:1.2em;--icon-height:1.2em;" href="//www.bilibili.com/video/BV1s44y1k7pK/" target="_blank" data-type="search" data-keyword="︎ı||ııı||ı|ıı||ı| 9”                                                          展开">ı||ııı||ı|ıı||ı| 9”                                                          展开</a>

**Comment by yiluoming** [comment]
你被骗了。

**Comment by Frieezed** [comment]
被诈骗多了，这链接都能背下来了

**Comment by haloged** [comment]
这个条约很重要，建议放在绳网必看

**Comment by Obstacle000** [comment]
可以的

**Comment by 15976724736** [comment]
很详细的跳跃，很清晰（我绝对不是坐忘道）

**Comment by Beichengyy796** [comment]
阿米诺斯

**Comment by FeiXiao-zm** [comment]
日日上一当，当当不一样

**Comment by Tragedysaver1024** [comment]
耍我啊混账

**Comment by Grey-Zero** [comment]
![Uploading 屏幕截图 2023-06-02 121516.png…]()

**Comment by Sushizhu** [comment]
新的协议更加具体人性，建议大家都看一下

**Comment by lizard315315** [comment]
非常棒

**Comment by Clockwise-26** [comment]
女娲！

**Comment by dominic121-dot** [comment]
我就知道

**Comment by niegong** [comment]
又被骗了

**Comment by GanlinSama** [comment]
又被骗了，这是今天第二次了（恼

**Comment by WLMRMZ** [comment]
特别详细，很感人，我都想哭了

**Comment by limbus666** [comment]
治安官已经出动去抓捕诈骗犯了，请各位不要慌张。另外，我是治安局的警探朱鸳，V我50，赠送反诈小技巧一份，谢谢各位市民配合。

**Comment by QingMingqwq** [comment]
确实，绳网不能什么都乱发，大家一定要仔细看看，遵守好协约qwq

**Comment by liugunshi** [comment]
没错，up说得对！

**Comment by githut48** [comment]
up,说的对

**Comment by sunnyt0wn** [comment]
b站以前的纯数字av号还是很好记，但现在有必要把这个bv号也记一下了.......

**Comment by zhangshshhhh** [comment]
怎么我逛绳网都能被骗

**Comment by nidegebijialaowang** [comment]
WTF

**Comment by Yanqing123469** [comment]
好好好

**Comment by maomiaonei** [comment]
这个协约真心希望大家都看看

**Comment by light814** [comment]
Σ(°Д°;

**Comment by mwk07** [comment]
怎么在这里都能被骗(@>@)

**Comment by wuzz1234567890** [comment]
6
